agr_java_core/src/main/java/org/alliancegenome/neo4j/view/ParalogBean.java

```diff
@@ -0,0 +1,67 @@
+package org.alliancegenome.neo4j.view;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonView;
+import lombok.Data;
+import org.alliancegenome.neo4j.entity.node.Gene;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@Schema(name="OrthologView", description="POJO that represents the Paralog view")
+@JsonPropertyOrder({"gene", "homologGene", "length", "similarity", "identity", "rank", "predictionMethodsMatched", "predictionMethodsNotMatched", "predictionMethodsNotCalled", "methodCount", "totalMethodCount"})
+public class ParalogBean implements Serializable {
```

Comment:
Bean... implies a "backing bean" from the java yester years. Plus this shouldn't be in the "view" package that is for JsonViews not regular objects.